Program.cs

```diff
@@ -27,80 +28,69 @@
 var api = new SpotifyApi();
 foreach (var token in tokens)

```

Comment:
Can we be sure that old tokens are noe present in the token store? Do we delete tokens if both the access token and refresh token are invalid?

Reply:
Also, since I've forgotten, why do we separate tokens and users in two stores instead of always looking at the user store and add tokens to the user?

Reply:
No, this app does not delete tokens. If access and refresh tokens are invalid/expired then it will just fail for that user and continue to next. Your tokens were actually not valid anymore (last i tested), so it skips your user/tokens.